### ISSUE TITLE
1.6: LedgerSMB::File's default initialisation of 'file_path' attribute incorrect

### ISSUE THREAD

**Issue by ehuelsmann** [ledgersmb/LedgerSMB#3631]
The attribute is documented to be a `Maybe[Str]`, but the initialization function returns a `File::Temp::Dir`.

**Comment by ehuelsmann** [comment]
Important backtrace:

```
Attribute (file_path) does not pass the type constraint because: Validation failed for 'Maybe[Str]' with value File::Temp::Dir=HASH(0x55c00fa74840) at  accessor LedgerSMB::File::file_path (defined at /srv/ledgersmb/bin/../lib/LedgerSMB/File.pm line 157) line 113 
 LedgerSMB::File::file_path('LedgerSMB::File=HASH(0x55c00fa74618)') called at /srv/ledgersmb/bin/../old/bin/io.pl line 12674  
 lsmb_legacy::print_form at /srv/ledgersmb/bin/../old/bin/io.pl line 11435 
 lsmb_legacy::print at /srv/ledgersmb/bin/../old/bin/old-handler.pl line 1586  
 lsmb_legacy::try {...}  at /usr/share/perl5/Try/Tiny.pm line 1037 
 eval {...} at /usr/share/perl5/Try/Tiny.pm line 948  
 Try::Tiny::try('CODE(0x55c00f5bf8c8)', 'Try::Tiny::Catch=REF(0x55c00ee123e0)') called at /srv/ledgersmb/bin/../old/bin/old-handler.pl line 1769 
 require old/bin/old-handler.pl at /srv/ledgersmb/bin/../lib/LedgerSMB/PSGI.pm line 15810  
 LedgerSMB::PSGI::try {...}  at /usr/share/perl5/Try/Tiny.pm line 10311 
```

**Comment by ehuelsmann** [comment]
Fixed by 5ae4d136
